recipes/z3/all/conandata.yml

```diff
@@ -1,30 +1,13 @@
 sources:
-  "4.8.8":
-    url: "https://github.com/Z3Prover/z3/archive/z3-4.8.8.tar.gz"
-    sha256: "6962facdcdea287c5eeb1583debe33ee23043144d0e5308344e6a8ee4503bcff"
+  "4.12.1":
+    url: "https://github.com/Z3Prover/z3/archive/refs/tags/z3-4.12.1.tar.gz"
+    sha256: "a3735fabf00e1341adcc70394993c05fd3e2ae167a3e9bb46045e33084eb64a3"
+  "4.11.2":
+    url: "https://github.com/Z3Prover/z3/archive/refs/tags/z3-4.11.2.tar.gz"
+    sha256: "e3a82431b95412408a9c994466fad7252135c8ed3f719c986cd75c8c5f234c7e"
   "4.10.2":
-    url: "https://github.com/Z3Prover/z3/archive/z3-4.10.2.tar.gz"
+    url: "https://github.com/Z3Prover/z3/archive/refs/tags/z3-4.10.2.tar.gz"
```

Comment:
Please, prefer the project's release instead of Github generated tars. We had an incident in the past with checksums, and the recommendation was always keep using the project release package when possible.

Reply:
There is no project release of the source code. I think. `https://github.com/Z3Prover/z3/archive/refs/tags/z3-4.10.2.tar.gz` is slightly better than `https://github.com/Z3Prover/z3/archive/z3-4.10.2.tar.gz`, but both are auto generated by github anyway.